proto/exocore/restaking_assets_manage/v1/tx.proto

```diff
@@ -88,9 +113,9 @@ message StakerSingleAssetOrChangeInfo {
     (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
     (gogoproto.nullable) = false
   ];
-  // wait_undelegation_amount_or_want_change_value is the amount that is waiting for undelegation
+  // wait_unbonding_amount_or_want_change_value is the amount that is waiting for undelegation
   // or the amount by which it can change.
-  string wait_undelegation_amount_or_want_change_value = 3
+  string wait_unbonding_amount_or_want_change_value = 3
```

Comment:
Still mixing two purposes in the same field? Better separate them.

Reply:
Yeah, I will refine them in the next commit.